config/packages/dev/monolog.yml

```
@@ -11,3 +11,5 @@ monolog:
         console:
             type:     console
             channels: ['!event', '!doctrine', '!event_api', '!business_event']
+        business_event:
+            type:     "null"
```

please change for simple quote and don't align the "null" with the other line.
```suggestion
            type: 'null'
```

---

👍 thanks for the review